learning_paths/visualising_architecture/2_c4_model.md

```diff
@@ -0,0 +1,26 @@
+# C4 Model
+The [C4 Model](https://c4model.com/) focusses on the visual communication and documentation of software architecture. It doesn’t present a formalised, standardised method to communicate software architecture, it does provide a collection of lightweight ideas and techniques that we've observed to be useful.
```

Comment:
I know focusses is a valid spelling but it always looks slightly odd. You can feel free to ignore this though.

```suggestion
The [C4 Model](https://c4model.com/) focuses on the visual communication and documentation of software architecture. It doesn’t present a formalised, standardised method to communicate software architecture, it does however provide a collection of lightweight ideas and techniques that we've observed to be useful.
```